.github/actions/build-engine/action.yaml

```diff
@@ -0,0 +1,109 @@
+# .github/actions/build-engine/action.yml
+name: ""
+description: "Build the engine with a specific gclient setup and config name"
+inputs:
+
+  gclient:
+    description: "name of gclient configuration to use"
+    required: true
+    
+  config_name:
+    description: "name of a valid et config"
+    required: true
+
+  engine_version:
+    description: "the engine version for the output assets"
+    required: true
+
+  gcp_credentials:
+    description: "the credentials for a GCP storage bucket for the outputs"
+    required: false
+    default: ''
+
+  gcp_bucket:
+    description: "the bucket to push builds to"
```

Comment:
Same here. And what if the user wants to upload somewhere else? Is that out of scope for now?